apps/judicial-system/backend/src/app/modules/subpoena/subpoena.service.ts

```diff
@@ -41,13 +44,29 @@ export class SubpoenaService {
     @InjectModel(Subpoena) private readonly subpoenaModel: typeof Subpoena,
     @InjectModel(Defendant) private readonly defendantModel: typeof Defendant,
     private readonly pdfService: PdfService,
+    private readonly messageService: MessageService,
     @Inject(forwardRef(() => PoliceService))
     private readonly policeService: PoliceService,
     @Inject(forwardRef(() => FileService))
     private readonly fileService: FileService,
     @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
   ) {}
 
+  private isSuccessfulServiceStatus(serviceStatus?: ServiceStatus) {
```

Comment:
There is a helper function that already does this in the lib in the defendant.ts file, did you decide not to use that on purpose? If so, why? 😊